production.Dockerfile

```diff
@@ -14,6 +14,7 @@ RUN yarn build
 # Bundle static assets with nginx
 FROM nginx:1.21.0-alpine as production
 ENV NODE_ENV production
+ENV REACT_APP_BACKEND_PUBLIC_URL https://clima.arpa.veneto.it
```

Comment:
We now have three dockerfiles, which seem to differ only in how they are getting these env variables - how about consolidating into a single `Dockerfile`, which has default values for these env values suitable for dev, and which can then be parametrized when running `docker build` via passing `--build-arg`s for them?

Reply:
Coping with three different `Dockerfiles` is also introducing some unneeded complexity to the CI workflow, which I'd try to avoid as much as possible